lib/src/main/kotlin/com/swmansion/starknet/crypto/NativeLoader.kt

```diff
@@ -62,8 +63,28 @@ internal object NativeLoader {
     private fun getLibPath(system: SystemType, architecture: String, name: String): String {
         return when (system) {
             SystemType.MacOS -> "/darwin/$name.dylib"
-            SystemType.Linux -> "/linux/$architecture/$name.so"
-            else -> throw UnsupportedPlatform(operatingSystem.name, architecture)
+            SystemType.Linux -> {
```

Comment:
maybe it would be possible to have a separate system type for android?

Reply:
Keep in mind that if Android app uses JAR (which is possible) then the path should still include linux. Your suggestion makes sense but it won't work in all possible cases.

Reply:
this will be if android is x86 right? or some other cases are possible as well?
if true, maybe we should also check the architecture for android?

let's think about it and discuss first, if the effort is not worth it it may be good idea to just leave it as is right now

Reply:
It turns out this changes is not needed - this logic is fine and we don't have to adjust it.